elements/pfe-navigation/documentation/implementation.md

```diff
@@ -0,0 +1,227 @@
+# Implementation
+
+> If there are any questions, issues, feel there's something missing, please contact the CPFED Group.
+
+## Adding scripts and styles
+Follow the [instructions from Patternfly Elements to get the javascript added to the page](https://patternfly.github.io/patternfly-elements/getting-started/).
+
+Add the fallback stylesheet, which covers non-JS users and initial page load nicities:
+```html
+<link rel="stylesheet" href="PATH/TO/pfe-navigation--lightdom.css" type="text/css">
+```
+
+## Adding HTML
+
+The bare minimum skeleton HTML is:
+
+```html
+<!-- These links should be directly after <body> -->
+<a href="#pfe-navigation" class="visually-hidden">Skip to navigation</a>
+<!-- !! Update anchor link to main/content -->
+<a href="#ADD-ID-TO-MAIN" class="visually-hidden">Skip to content</a>
+
+<pfe-navigation id="pfe-navigation">
+  <nav class="pfe-navigation" aria-label="Main Navigation">
+    <div class="pfe-navigation__logo-wrapper" id="pfe-navigation__logo-wrapper">
+      <a href="/" class="pfe-navigation__logo-link">
+        <!-- !! Update logo src -->
+        <img
+          class="pfe-navigation__logo-image"
+          src="assets/redhat--reverse.svg" width="400" alt="Redhat"
+        />
+      </a>
+    </div>
+    <ul class="pfe-navigation__menu" id="pfe-navigation__menu">
+      <li class="pfe-navigation__menu-item">
+        <a href="#LINK-TO-CONTENT" class="pfe-navigation__menu-link">
+          Menu Link 1
+        </a>
+      </li>
+
+      <li class="pfe-navigation__menu-item">
+        <a href="#LINK-TO-CONTENT" class="pfe-navigation__menu-link">
+          Menu Link 2
+        </a>
+      </li>
+
+      <li class="pfe-navigation__menu-item">
+        <a href="#LINK-TO-CONTENT" class="pfe-navigation__menu-link">
+          Menu Link 3
+        </a>
+      </li>
+
+      <li class="pfe-navigation__menu-item">
+        <a href="#LINK-TO-CONTENT" class="pfe-navigation__menu-link">
+          Menu Link 4
+        </a>
+      </li>
+    </ul>
+
+  </nav>
+</pfe-navigation>
+```
+
+> Unfortunately we need to make sure all of the id's and classes are correct on this level for fall back styling, functionality may break if classes or id's are missing or incorrect.
+
+### Adding dropdowns to the menu
+To add a dropdown, add the following markup inside of an `<li class="pfe-navigation__menu-item">`, but after the `<a>`, like this:
+
+```html
+<li class="pfe-navigation__menu-item">
```

Comment:
Instead of relying on classes to style the dropdown, let's create a navigation item component like we had before and support variants on them to style the widths appropriately.